tests/test_modelgen.py

```diff
@@ -49,51 +52,38 @@ def _generate_train_data(self, x_shape, nr_classes):
         y_train = np.random.randint(0, 1, size=(1, nr_classes))
         return X_train, y_train
 
-    def test_regularization_is_float(self):
-        """ Regularization should be a float. """
-        reg = modelgen.get_regularization(0, 5)
-        assert type(reg) == np.float
-
-    def test_regularization_0size_interval(self):
-        """ Regularization from zero size interval [2,2] should be 10^-2. """
-        reg = modelgen.get_regularization(2, 2)
-        assert reg == 0.01
-
-    def test_base_hyper_parameters_reg(self):
-        """ Base hyper parameter set should contain regularization. """
-        hyper_parameter_set = modelgen.generate_base_hyper_parameter_set(low_lr=1,
-                                                                         high_lr=4,
-                                                                         low_reg=1,
-                                                                         high_reg=3)
-
-        assert 'regularization_rate' in hyper_parameter_set.keys()
```

Comment:
This has been moved to tests/test_base_hyperparameter_generator.py
Modified only to remove ```modelgen.``` from the method calls